backend/de.metas.contracts/src/main/java/de/metas/contracts/modular/log/ModularContractLogEntry.java

```diff
@@ -117,6 +118,12 @@ public class ModularContractLogEntry
 	@Nullable
 	Integer storageDays;
 
+	@Nullable
+	Integer interestDays;
+
+	@Nullable
+	Percent interestPercent;
```

Comment:
wondering if it makes sense to introduce ModularContractLogEntry.Interest class that contains interestDays and interestPercent.

In this way you could use Interest class in request, response or wherever is needed.

benefits:
* similar data is grouped in one class
* maybe you will make them not null in the Interest class (better validation)
* in case you need to extend, you just have to add more fields to Interest class

Reply:
As this field is used together with different fileds on purchase (just not propagated there yet) and sales contract I wouldn't do it here